content/en/tracing/trace_collection/custom_instrumentation/dd_libraries/cpp.md

```diff
@@ -23,100 +23,115 @@ further_reading:
 If you have not yet read the setup instructions, start with the <a href="https://docs.datadoghq.com/tracing/setup/cpp/">C++ Setup Instructions</a>.
 </div>
 
-## Add tags
+## Creating spans
 
-Add custom [span tags][1] to your [spans][2] to customize your observability within Datadog. The span tags are applied to your incoming traces, allowing you to correlate observed behavior with code-level information such as merchant tier, checkout amount, or user ID.
-
-C++ tracing uses "common tags". These tags can be sourced from both [Datadog specific tags][3] or [OpenTracing tags][4], and included like this:
+### Manually instrument a method
 
 ```cpp
-#include <opentracing/ext/tags.h>
-#include <datadog/tags.h>
+{
+  // Create a root span for the current request.
+  auto root_span = tracer.create_span();
```

Comment:
Can we include the `include` statements for all examples? Ideally, the docs are close to runnable

Reply:
I prefer to keep the snippet focused on what we want to demonstrate, so as not to distract the viewer.